### DISCUSSION TITLE
Development status of this billing software?

### DISCUSSION THREAD

**Discussion by DewlanceHosting** [Katamaze/Leap#22] [General]
Hi,

Katamaze, why did you stop the development of your Leap billing software? As far as I know, you are a developer, and if you focus on it, it could become the most widely used open-source billing software.

Please continue the development of Leap. Don’t k-ill it.

Thanks.

**Comment by teklangroup** [comment]
> Hi,
> 
> Katamaze, why did you stop the development of your Leap billing software? As far as I know, you are a developer, and if you focus on it, it could become the most widely used open-source billing software.
> 
> Please continue the development of Leap. Don’t k-ill it.
> 
> Thanks.

Offer to sponsor his time if you want it? It's all well and good saying build this for free for me but it's not always viable when people have to earn a living

**Comment by andrekutianski** [comment]
I'm working on a new platform called DynaBill, which is meant to be a great alternative to WHMCS. It's still in the early stages, but some people are backing me up now. While there's no finished product yet, we're working hard and hope to release an early version for people to test soon.

I'd appreciate your support. Even a simple follow or star on the project would mean a lot to us and keep us motivated to keep working on it in our spare time.

* GitHub profile: https://github.com/dynabill
* Project Roadmap: https://github.com/dynabill/roadmap
* Website: https://dynabill.cloud/

Best.

**Comment by KaiSchwarz-cnic** [comment]
@andrekutianski I offered my help to Project Leap, so I will to DynaBill. I can help with Domain Registrar Modules for CentralNic Reseller (WHMCS Partner), internet.bs and TPP Wholesale. Ping [me](https://www.linkedin.com/in/kai-schwarz-msc/) at LinkedIn if that is of interest and time is come.
